reports/GoodsAndServiceTax/BaseGSTR.js

```diff
@@ -46,7 +48,7 @@ class BaseGSTR {
     row.invNo = ledgerEntry.name;
     row.invDate = ledgerEntry.date;
     row.rate = 0;
-    row.inState = true;
+    row.inState = gstin.substring(0, 2) === posMap[row.place];
```

Comment:
we'll update this also once we add a dedicated place of supply field to the table.

let's also check how EN has done it.